.github/workflows/deploy-to-s3.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy To S3
+
+on:
+  push:
+    branches: [ official ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.CI_CD_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.CI_CD_AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wrc
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Is this line necessary, and where does it originally come from?

Reply:
I removed it